File: 46주차/가장 큰수.js
function solution(numbers) {
  var answer = "5";
  answer = numbers
    .map((c) => String(c))
    .sort((a, b) => b + a - (a + b))
    .join("");

  return answer[0] === "0" ? "0" : answer; // 000..이 될 경우를 생각해서 0 반환.
}

File: 47주차/가장큰수.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

bool compare(const string &a, const string &b){// 610, 106중 610이 더 크므로, 더 앞으로 정렬 이란뜻
    return (b+a < a+b) ;

}

string solution(vector<int> numbers) {
    string answer = "";
    vector<string> s;
    for (int num:numbers){
        s.push_back(to_string(num));
    };
    sort(s.begin(),s.end(),compare);
    
    for (string v: s){
        answer += v;
    }
    if (answer[0] == '0')//만약 [0,0,0,,] 와 같은 배열이라면 0.
        answer = "0";
    return answer;
}